tuyaha/devices/cover.py

```diff
@@ -2,27 +2,25 @@
 
 
 class TuyaCover(TuyaDevice):
+
     def state(self):
         state = self.data.get("state")
         return state
 
     def open_cover(self):
         """Open the cover."""
-        success, _response = self.api.device_control(self.obj_id, "turnOnOff", {"value": "1"})
-
-        if success:
-            self.data["state"] = True
+        if self._control_device("turnOnOff", {"value": "1"}):
+            self._update_data("state", 1)
```

Comment:
`state` was `True`/`False` previously. Now it's `1`, `2`, `3`. Can you explain?

Reply:
True and False was changed in previous PR, but this is not correct. Real state is 1,2,3, indeed cover do not work anymore properly in HA after last PR